src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -61,7 +61,7 @@ struct KeyValueViewModel {
         } else if let signatureParameter = operation.signatureParameter(for: param.name) {
             // value is referring a signautre parameter, no need to wrap as String
             self.paramName = param.name
-            self.optional = signatureParameter.required ?? true
+            self.optional = !signatureParameter.required
```

Comment:
What is the `!` for here? I've not seen that syntax before, but it seems like a "crash if nil" thing that we should avoid.